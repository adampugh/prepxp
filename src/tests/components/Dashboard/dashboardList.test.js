import React from "react";
import { shallow } from "enzyme";

import DashboardList from "../../../components/Dashboard/dashboardList";
import user from "../../fixtures/userLists";

describe("<DashboardList />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<DashboardList list={user.lists[0]} />);
        expect(wrapper).toMatchSnapshot();
    });
});