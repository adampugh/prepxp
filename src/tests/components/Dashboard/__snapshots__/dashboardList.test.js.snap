// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardList /> should match snapshot 1`] = `
<div
  className="dashboard__grid__lists__item"
>
  <div>
    <Link
      replace={false}
      to={
        Object {
          "pathname": "/list/start",
          "state": Object {
            "list": Object {
              "id": "234",
              "questions": Array [
                Object {
                  "answer": "JavaScript is a language for the web.",
                  "question": "What is the difference between let and var?",
                },
                Object {
                  "answer": "Brendan Eich",
                  "question": "Who created JavaScript?",
                },
              ],
              "title": "JavaScript Developer",
            },
          },
        }
      }
    >
      <h2>
        JavaScript Developer
      </h2>
    </Link>
    <p>
      2
       Questions
    </p>
  </div>
  <div>
    <Link
      replace={false}
      to={
        Object {
          "pathname": "/list/edit",
          "state": Object {
            "list": Object {
              "id": "234",
              "questions": Array [
                Object {
                  "answer": "JavaScript is a language for the web.",
                  "question": "What is the difference between let and var?",
                },
                Object {
                  "answer": "Brendan Eich",
                  "question": "Who created JavaScript?",
                },
              ],
              "title": "JavaScript Developer",
            },
          },
        }
      }
    >
      <h2>
        <i
          className="fas fa-edit"
        />
      </h2>
    </Link>
    <h2
      onClick={[Function]}
    >
      <i
        className="fas fa-trash-alt"
      />
    </h2>
  </div>
</div>
`;
